packages/broker/package.json

```diff
@@ -22,14 +22,15 @@
     "test": "jest test/unit test/integration && npm run test-sequential",
     "test-unit": "jest test/unit",
     "test-sequential": "jest --forceExit --maxWorkers=1 test/sequential # always run sequential tests with maxWorkers=1",
-    "test-integration": "jest --forceExit test/integration && npm run test-sequential"
+    "test-integration": "jest --testPathIgnorePatterns='test/integration/plugins/operator/' --forceExit test/integration && npm run test-sequential",
```

Comment:
It would be good to run all integration tests with `test-integration`.

Is there a need for this change as we can run single test or subset of tests from command line with `npx`:
`npx jest test/integration/plugins/operator/OperatorPlugin.test.ts`

Reply:
Removed the split.